src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class Ball {
+    public enum Colors{
+        red,
+        green,
+        yellow,
+        orange,
+        purple
+    }
+    /*
+    в задании сказано, что color & number = поля. У меня это методы. Можно сделать их полями, а методы добавить в классе
+    лоторея. Но я хз будет ли так лучше, ведь актуальный код менее об'ёмный. Но тогда можно было бы сделать поля приватными
+    но зачем))))))) мы туда ниче не вписываем
+    */
+    public String color(){
+        ColorSupplier supplier = new ColorSupplier();
+        return supplier.getRandomColor();
+    }
+    public String number() {
+        NumberSupplier supplier = new NumberSupplier();
+        return String.valueOf(supplier.getRandomNumber());
+    }
+    @Override
+    public String toString() {
+        String result = "You received the following ball: " + color() + ",\n" + number();
+        System.out.println(result);
+        return null;
```

Comment:
never return null. instead of this, just return that string
```suggestion
        return "You received the following ball: " + color() + ",\n" + number();
```